src/main/java/org/radarcns/management/service/mapper/ProjectMapper.java

```diff
@@ -28,6 +29,8 @@
 
     MinimalProjectDetailsDTO projectToMinimalProjectDetailsDTO(Project project);
 
+    MinimalProjectDetailsDTO projectDTOToMinimalProjectDetailsDTO(ProjectDTO projectDTO);
```

Comment:
It is a bit unconventional  to map one DTO to another DTO in `Mapstruct`. Why do we need this?

Reply:
It seems to be unused, possibly a leftover from an earlier implementation, agreed it should be removed.